jmclient/jmclient/wallet.py

```diff
@@ -181,18 +208,52 @@ def add_utxo(self, txid, index, path, value, mixdepth):
 
         self._utxo[mixdepth][(txid, index)] = (path, value)
 
+    def is_disabled(self, txid, index):
+        if self._utxo_meta is None:
+            return False
+        if (txid, index) not in self._utxo_meta.keys():
+            return False
+        if b'disabled' not in self._utxo_meta[(txid, index)]:
+            return False
+        if not self._utxo_meta[(txid, index)][b'disabled']:
+            return False
+        return True
```

Comment:
shouldn't this return `self._utxo_meta[(txid, index)][b'disabled']`?

Reply:
Ah yes, that's a bit silly. But I see now that this probably arose out of something I was uncertain of while coding this: as I understand it we use bencoder to persist this stuff to disk, and since we have a boolean here that I'd like to store in python as True/False, should this field be boolean or should we have it as 0/1?

Reply:
Good point, bencoder.pyx will indeed convert this to 0/1. Maybe a `return bool(…)`?